src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -27,20 +36,22 @@ public async Task<IActionResult> QualificationQuestion(string accountId, Qualifi
 
             if (viewModel.HasTakenOnNewApprentices.Value)
             {
-                return RedirectToAction("SelectApprenticeships");
+                return RedirectToAction("SelectApprenticeships", new { accountId });
             }
 
-            return RedirectToAction("CannotApply");
+            return RedirectToAction("CannotApply", new { accountId });
```

Comment:
Is the account id needed here, as there's no corresponding param in the controller action?

Reply:
Yes, it's the hashed id so I'll rename the parameter.  It's needed for the routing engine as it's in the controller path.

Reply:
Can't rename is, that also breaks the routing engine!